### ISSUE TITLE
Supremacist Ideology causes -30 towards Player Faction

### ISSUE THREAD

**Issue by Pingger** [Pingger/Empire-Mod#2]
Discovered by Discord User: clementine

**Comment by Pingger** [comment]
Commit needs testing

**Comment by FalconElaris** [comment]
I can test this. Which branch is it? v1.3compatibility?

**Comment by FalconElaris** [comment]
The +200 in the code seems to set the goodwill at +100, but the supremacist Ideology still lowers value to player faction. I think this is actually a good thing because it's thematically appropriate. If a settlement holds the supremacist ideology, wouldn't they think that they're supreme as a unit? I've attached a screenshot of my test's faction goodwill at start. I advanced time by 2 seasons, and the faction relations stayed at +100.
![FactionPlayer_relations_at_setup](https://user-images.githubusercontent.com/3372722/128597332-d20e4ab5-0a92-47ff-a734-e0fe0d071315.PNG)
